http/src/main/resources/reference.conf

```diff
@@ -45,6 +43,15 @@ dataproc {
   deployWelderCutoffDate = "2019-08-01"
 }
 
+image {
```

Comment:
I like this; I was thinking of further breaking out a `welder { }` in my PR with `deployWelderLabel`/ `updateWelderLabel` etc.